src/curve/stable/StablePair.sol

```diff
@@ -52,8 +52,8 @@ contract StablePair is ReservoirPair {
     uint64 private lastInvariantAmp;
 
     constructor(address aToken0, address aToken1) ReservoirPair(aToken0, aToken1, PAIR_SWAP_FEE_NAME) {
-        MINT_BURN_LOGIC = factory.deploy(ConstantsLib.MINT_BURN_KEY, aToken0, aToken1);
-        require(MINT_BURN_LOGIC != address(0), "SP: MINT_BURN_DEPLOYMENT_FAILED");
+        MINT_BURN_LOGIC = factory.read("SP::STABLE_MINT_BURN").toAddress();
+        require(address(MINT_BURN_LOGIC).code.length > 0, "SP: MINT_BURN_NOT_DEPLOYED");
```

Comment:
This should become a constant address instead of a key. We only used the key because we needed to deploy the contract's bytecode from a key. However, now that we have a single contract the address should be constant across all networks/deployments/stable-pairs